copying.md

```diff
@@ -153,6 +153,7 @@ _the openage authors_ are:
 | Astitva Kamble              | askastitva                  | astitvakamble5 à gmail dawt com                   |
 | Haoyang Bi                  | AyiStar                     | ayistar à outlook dawt com                        |
 | Michael Seibt               | RoboSchmied                 | github à roboschmie dawt de                       |
+| Nikhil Ghosh                | NikhilGhosh75               | nghosh606 à gmail dawt com
```

Comment:
I think the CI fails because you are missing an `|` at the end of the table. Then the mailmap entry is also not necessary.